.github/workflows/test-promot.yaml

```diff
@@ -0,0 +1,34 @@
+---
+name: test-promot
+
+on: workflow_dispatch
+
+jobs:
+  test-prolong:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: echo SUCCESS > .final_status
+      - name: Artifacts Upload
+        id: artifacts-upload
+        uses: ./
+        with:
+          url: ${{ secrets.ARTIFACTS_URL }}
```

Comment:
Let's make use of tomorrow session to make a peer coding of setting up a fake env to test the promotion.